web3-functions/pyth-oracle-w3f-priceIds/pythAbi.ts

```diff
@@ -0,0 +1,471 @@
+export const pythAbi = [
```

Comment:
Can you use the ABI from the SDK? see typescript installation instructions here https://docs.pyth.network/evm/sdks